scripts/cron-test.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -vx
+
+echo "Checking node version..."
+node --version | grep v8
+if [[ $? != 0 ]] ; then
+    # Don't run the tests if node version < 8
```

Comment:
How come?

Reply:
We use async/await in our tests. Also since we use strict mode in the tests, I also got the following error in older node versions - 
`SyntaxError: Block-scoped declarations (let, const, function, class) not yet supported outside strict mode`
Here are the failed runs - https://travis-ci.org/okta/okta-sdk-nodejs/builds/368896186

Reply:
Ah. We should probably update our tests to not use `async await` or change our test versions of **node** in our `travis.yml` file.

@robertjd Thoughts?

Reply:
We do need to continue supporting >=4.8.3 until our next major rev, when we can drop 4x and go up to 6x support.  As such we should be testing on all the versions we support.  Currently we aren't testing on any (bad).

I see a few ways out of this problem:
* Update tests to not use async
* Transform tests at runtime with babel?
* Separate out IT tests (and allow them to use async) from unit tests, forcing unit tests to stick to our minimum supported node version.  Then, run unit tests against all supported node versions, and only run IT tests on a modern node version.